src/lib/icon/icon-registry.ts

```diff
@@ -201,7 +201,7 @@ export class MatIconRegistry {
       return observableOf(cloneSvg(cachedIcon));
     }
 
-    return RxChain.from(this._loadSvgIconFromConfig(new SvgIconConfig(url)))
+    return RxChain.from(this._loadSvgIconFromConfig(new SvgIconConfig(safeUrl)))
```

Comment:
While we're at it, it might be a good idea to rename the `url` variable above to something along the lines of `sanitizedUrl` to avoid confusion in the future.

Reply:
Really the _type_ should enforce passing the right thing around; I want to check with Martin about adding a brand to safe types (since they're empty they match everything)